src/config.rs

```diff
@@ -51,6 +51,7 @@ pub struct CameraConfig {
     pub username: String,
     pub password: Option<String>,
 
+    // no longer used, but still here so we can warn users:
     pub timeout: Option<Duration>,
```

Comment:
Rather than a warning what about a validation error and it exits? Or am I being too mean..

Reply:
Since it's otherwise harmless to have there, I'd say I prefer a graceful migration. Now if it was an "action required" behavior change, I would agree.

Reply:
Fair enough, since its undocumented anyways I guess we can just do it that with a warning